core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -206,6 +210,10 @@ private FileSystemContext(ClientContext ctx) {
     LOG.info("Created filesystem context with id {}. This ID will be used for identifying info "
             + "from the client, such as metrics. It can be set manually through the {} property",
         mAppId, PropertyKey.Name.USER_APP_ID);
+
+    mWorkerGroup = NettyUtils.createEventLoop(NettyUtils.getUserChannel(ctx.getConf()),
+        ctx.getConf().getInt(PropertyKey.USER_NETWORK_NETTY_WORKER_THREADS),
+        String.format("alluxio-client-nettyPool-%s-%%d", mAppId), true);
```

Comment:
Is `%%d` the correct formatting?

Reply:
Passing `%%d` escapes the `%` in java's string.format